src/pages/home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+//requisito 2
+
+class Home extends React.Component {
+  render () {
+    return (
+      <div>
+        <input className='App' type='text' />
```

Comment:
Correção apontada pelo ESLint
```suggestion
        <input className="App" type="text" />
```